Northstar.CustomServers/mod/scripts/vscripts/mp/_base_gametype_mp.gnut

```diff
@@ -398,6 +403,23 @@ void function PostDeathThread_MP( entity player, var damageInfo ) // based on ga
 	}
 }
 
+// idk if this is a good delay or if it matches vanilla
+void function ForceRespawnMeSignalAfterDelay( entity player, int delay = 5 )
+{
+	player.EndSignal( "RespawnMe" )
+
+	if( player.IsWatchingKillReplay() )
+		player.WaitSignal( "KillCamOver" )
+
+	wait delay
+
+	if (!IsValid(player))
```

Comment:
this should be replaced with a `player.EndSignal( "OnDestroy" )` at the start of function.
EndSignals are more reliable.